recipes/bzip2/all/conanfile.py

```diff
@@ -1,8 +1,10 @@
 import os
 import textwrap
-from conans import ConanFile, CMake, tools
+from conans import ConanFile, tools
```

Comment:
Should we start to try to replace all the `tools.xxx` also?  (in general all the conans imports)